### PR TITLE
Documentation for person-detection-0106

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
@snosov1 @Ilya-Krylov

**Comment by Ilya-Krylov** [comment]
@eizamaliev

File: models/intel/person-detection-0106/description/person-detection-0106.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## Performance
+
+## Inputs
+
+Name: `input`, shape: [1x3x800x1344] - An input image in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
```
Comment (eizamaliev) [inline models/intel/person-detection-0106/description/person-detection-0106.md:38]: Please, remove indentation

Comment (druzhkov-paul) [reply models/intel/person-detection-0106/description/person-detection-0106.md]: Removed here and in readme files for some other models as well. Please take a look.

**Pull request merged**